frontend/app/views/fragments/form/cardDetail.scala.html

```diff
@@ -1,179 +0,0 @@
-                            </p>
-                        </div>
-                    </div>
-                </div>
-            </div>
-
-            <div class="form-field">
-                <label class="label" for="cc-exp-month"><span>Expiry</span></label>
-                <select data-stripe="exp-month"
-                        id="cc-exp-month"
-                        data-validation="validCreditCardMonth"
-                        class="js-credit-card-exp-month">
-                    <option selected="selected">Month</option>
-                    @for(month <- selectDates.validCardMonths){
-                    <option value="@month">@month</option>
-                    }
-                </select>
-                <span> / </span>
-                <select data-stripe="exp-year"
-                        id="cc-exp-year"
-                        data-validation="validCreditCardYear"
-                        class="js-credit-card-exp-year">
-                    <option selected="selected">Year</option>
-                    @for((year, formattedYear) <- selectDates.validCardYears){
-                    <option value="@year">@formattedYear</option>
-                    }
-                </select>
-                @fragments.form.errorMessage("Sorry, the expiry date that you have entered is invalid. Please check and re-enter.")
-            </div>
-        } { card =>
-            <div class="form-field u-cf">
-                <div class="card-summary">
-                    <div class="card-summary__type">
-                        <i class="sprite-card sprite-card--small sprite-card--@(card.issuer)"></i>
-                    </div>
-                    <div class="card-summary__digits">
-                        •••• •••• •••• •••• @card.last4
-                    </div>
-                </div>
-            </div>
-        }
-
-        <div class="bottom-credit-card-description">
-            <div class="bottom-credit-card-description--note">
-                <h3 class="credit-card-note__header">First payment</h3>
-                <p class="credit-card-note__text">will be taken today under the name Guardian News & Media.</p>
-            </div>
-            <div class="bottom-credit-card-description--note js-ongoing-card-payments"">
-                <script type="text/javascript">
-                    guardian.membership.checkoutForm.ongoingCardPayments = {
-                        annual: '@Dates.dayInMonthWithSuffixAndMonth()',
-                        monthly: '@Dates.dayInMonthWithSuffix()'
-                    };
-                </script>
-            </div>
-            <div class="bottom-credit-card-description--note">
-                <h3 class="credit-card-note__header">Cancellation</h3>
-                <p class="credit-card-note__text">You can cancel your membership online through your profile page.</p>
-            </div>
-        </div>
-
-    </div>
-</fieldset>
```

Comment:
It's odd that this isn't in use. @AWare is it possible that this was the form used prior to the introduction of Stripe Checkout?

@JustinPinner with this file gone is it possible that [this listeners file](https://github.com/guardian/membership-frontend/blob/3e93a9826d3091c32059cf84e45fb472eea7c268/frontend/assets/javascripts/src/modules/form/payment/listeners.js) is also now redundant?

Reply:
I would also guess pre-Stripe checkout - perhaps @AWare or @paulbrown1982 can confirm as they're probably the most familiar with this code?

Reply:
This was the old form. It should be deleted.